src/sentry/integrations/github/client.py

```diff
@@ -58,8 +58,9 @@ def __repr__(self) -> str:
 
 
 class GithubProxyClient(IntegrationProxyClient):
+    integration: RpcIntegration | Integration
```

Comment:
I think this is due to it being *expected* from a subclass -- but it turns out there's only one subclass!  so let's merge `GithubProxyClient` and the class below and then I think this annotation may be able to be removed?